Platform/CommonBoardPkg/Include/Library/BoardSupportLib.h

```diff
@@ -72,4 +88,34 @@ CheckStateMachine (
   IN FW_UPDATE_STATUS    *pFwUpdStatus
   );
 
+/**
+  Patch ACPI CPU Pss Table
+
+  This function will patch PSS table. Caller MUST guarantee valid table address.
+
+  @param[in,out]  PssTableAddr      Pointer to PSS Table to be updated
+  @param[in]      TurboBusRatio     Turbo bus ratio
+  @param[in]      MaxBusRatio       Maximum bus ratio
+  @param[in]      MinBusRatio       Mimimum bus ratio
+  @param[in]      PackageMaxPower   Maximum package power
+  @param[in]      PackageMinPower   Minimum package power
+  @param[in]      GetRelativePower  A func pointer to get relative power
+  @param[in]      DoListAll         List all from LFM to Turbo
+
+  @retval         EFI_SUCCESS       Patch done successfully
+  @retval         others            Error occured during patching the table
+
+**/
+EFI_STATUS
+AcpiPatchPssTable (

```

Comment:
Can we pass in a data structure pointer instead ?   
It is too many parameters to pass.

Reply:
Thanks for the feedback. Let me make a following up PR.